src/core/Akka.Persistence.TCK/Akka.Persistence.TCK.csproj

```diff
@@ -16,6 +16,10 @@
     <ProjectReference Include="..\Akka.Streams.TestKit\Akka.Streams.TestKit.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="$(FluentAssertionsVersion)" />
```

Comment:
we should probably just add a dependency to this in the TestKit itself - but that's an issue for another day